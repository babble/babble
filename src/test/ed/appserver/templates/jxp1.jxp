<div>
  <%= 5 + 6 %>
</div>

<div class=123>
  <%= 5 + 6 %>
</div>
