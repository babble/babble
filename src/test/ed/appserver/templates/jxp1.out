<div>
        11
</div>

<div class=123 >
        11
</div>
